lib/cambiatus/social/social.ex

```diff
@@ -0,0 +1,130 @@
+defmodule Cambiatus.Social do
+  @moduledoc """
+    The Social context. Handles everything related to news.
+  """
+
+  import Ecto.Query
+
+  alias Cambiatus.Commune
+  alias Cambiatus.Repo
+  alias Cambiatus.Social.News
+  alias Cambiatus.Social.NewsReceipt
+  alias Cambiatus.Social.NewsVersion
+  alias Cambiatus.Workers.ScheduledNewsWorker
+  alias Ecto.Multi
+
+  @spec data :: Dataloader.Ecto.t()
+  def data(params \\ %{}), do: Dataloader.Ecto.new(Repo, query: &query/2, default_params: params)
+
+  def query(News, _params) do
+    News
+    |> order_by([n], desc: n.inserted_at)
+  end
+
+  def query(queryable, _params) do
+    queryable
+  end
+
+  def get_news(news_id) do
+    News
+    |> Repo.get(news_id)
+  end
```

Comment:
hmm, this call only call another function. I know alias can be useful, but here... Idk... At least we can make it shorter without the pipe:

```suggestion
  def get_news(news_id), do: Repo.get(News, news_id)
```